code/modules/detectivework/microscope/microscope.dm

```diff
@@ -1,142 +1,40 @@
 //microscope code itself
-/obj/machinery/microscope
+/obj/machinery/forensic/microscope
```

Comment:
You need to either set base_type on this one or not set it on the parent; currently this will search its parent for an appropriate circuitboard building it.

Reply:
it's set on both subtypes to their own type